src/server/examples/README.md

```diff
@@ -0,0 +1,17 @@
+# Example Projects
```

Comment:
Ah, yeah, a link should be fine in that case. I think of the docs as mostly a user-level thing rather than dev, so I didn't think to look there. Also never would have guess there's an "export role" option (but only in a hidden menu?)

Reply:
Yeah, updating the format of the examples has been on my to-do list for a _long_ time but it has always been such a low priority that it keeps getting pushed off :)